segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/ContainerRecoveryUtils.java

```diff
@@ -278,4 +392,42 @@ private static void validateContainerIds(Map<Integer, DebugStreamSegmentContaine
             }, executor);
         }, executor);
     }
+
+    /**
+     * This method creates a back up segment of container metadata segment and its attribute segment for each
+     * container Id. The original metadata segments and its attribute segments are deleted and the back up copy
+     * of original metadata segments are stored in a map and returned.
+     *
+     * @param storage                   A {@link Storage} instance to get the segments from.
+     * @param containerCount            The number of containers for which renaming of container metadata segment and its
+     *                                  attributes segment has to be performed.
+     * @param executorService           A thread pool for execution.
+     * @return                          A Map of Container Ids to new container metadata segment names.
+     * @throws InterruptedException     If the operation was interrupted while waiting.
+     * @throws TimeoutException         If the timeout expired prior to being able to complete the operation.
+     * @throws ExecutionException       When execution of the opreations encountered an error.
+     */
+    public static Map<Integer, String> getBackUpMetadataSegments(Storage storage, int containerCount, ExecutorService executorService)
```

Comment:
Your Javadoc says "creates a backup" yet this method is named `get`. Consider renaming to `createBackupMetadataSegments`

Reply:
Done.